### DISCUSSION TITLE
Times jump on "Apply to all Dates"

### DISCUSSION THREAD

**Discussion by ARogovskyy** [lukevella/rallly#1061] [General]
**Describe the bug**
With 2 time options per day and multiple dates, using "apply to all dates" changes the times for the first date (it should only change the times for the other dates).

**To Reproduce**

1. Create a new Poll
2. Add two dates and activate "specify times"
3. Add two time slots 12:15-13:45 and 14:00-15:00 to the first date
4. In the context menu of the first date, click on "apply to all dates"

**Expected behavior**
Both dates have 2 timeslots, 12:15-13:45 and 14:00-15:00

**Actual Behavior**
The first time slot jumps to 12:15-13:15

**Screenshots**
Before clicking "apply to all dates":
![image](https://github.com/lukevella/rallly/assets/48906414/009d72a8-7057-4c21-8b75-3f2b75f258aa)

After:
![image](https://github.com/lukevella/rallly/assets/48906414/259318c5-b12c-4468-a6eb-98191f630bd7)

**Desktop (please complete the following information):**

- OS: Arch Linux
- Browser Firefox 123
- Version 

**Additional context**
It seems like rallly tries to enforce a consistent time duration for the slots.

**Comment by lukevella** [comment]
This is actually the expected behavior. When you select "Apply to all dates", all the dates are give the same time options. All other options are removed. To me this makes sense but I suppose you can make an argument for both. I think I significant revamp is due and I'll be working on this soon. Happy to gather feedback on how users think this should work.

Moving to discussions.
